### PR TITLE
docs: update response format page #563

### LINKED ISSUE #563: Docs: Rework "Response format"
Do we need it at all? Should be redundant with the Swagger UI.

If we keep it, fix it. It's probably not up to date with LAPIS 2.

### PR DESCRIPTION
**Pull request by JonasKellerer**
resolves #563 

## PR Checklist
~~- [ ] All necessary documentation has been adapted.~~
~~- [ ] The implemented feature is covered by an appropriate end-to-end test.~~

### CODE REVIEW

File: lapis2-docs/src/content/docs/concepts/response-format.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+![Example nucleotide mutation response](/images/docs/concepts/example_nucleotide_mutation_response_200.png)
+
+At the top, you'll find a plain text description of the response.
+The middle section displays a JSON format response example.
+For detailed information about the response data, click on 'Schema'.
+This reveals the response structure and descriptions for each field, accessible by clicking the three dots next to them.
+
+![Example nucleotide mutation response schema](/images/docs/concepts/example_nucleotide_mutation_schema.png)
+
+Fields marked with an asterisk (\*) are always included in the response; unmarked fields may be omitted.
```
Comment (fengelniederhammer) [inline lapis2-docs/src/content/docs/concepts/response-format.mdx:29]: Not an issue of this PR, but I wonder why the image above shows data version as "required", but request id, mutation, proportion and count seem to be optional...

File: lapis2-docs/src/content/docs/concepts/response-format.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Endpoints typically support returning data in JSON, [CSV](https://en.wikipedia.org/wiki/Comma-separated_values) or
+[TSV](https://en.wikipedia.org/wiki/Tab-separated_values), with JSON being the default.
+Genomic sequences (such as those from `unalignedNucleotideSequences`, `alignedAminoAcidSequences`, etc.) are provided in the
+[FASTA format](https://en.wikipedia.org/wiki/FASTA_format).
+
+## Example
+
+To understand the response from the `nucleotideMutation` endpoint, refer to the relevant section in the Swagger UI:
+
+![Example nucleotide mutation response](/images/docs/concepts/example_nucleotide_mutation_response_200.png)
```
Comment (fengelniederhammer) [inline lapis2-docs/src/content/docs/concepts/response-format.mdx:20]: This picture is missing.

File: lapis2-docs/src/content/docs/concepts/response-format.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+At the top, you'll find a plain text description of the response.
+The middle section displays a JSON format response example.
+For detailed information about the response data, click on 'Schema'.
+This reveals the response structure and descriptions for each field, accessible by clicking the three dots next to them.
+
+![Example nucleotide mutation response schema](/images/docs/concepts/example_nucleotide_mutation_schema.png)
+
+Fields marked with an asterisk (\*) are always included in the response; unmarked fields may be omitted.
+
+{/* prettier-ignore */}
```
Comment (fengelniederhammer) [inline lapis2-docs/src/content/docs/concepts/response-format.mdx:31]: Do we need this?

**Review by fengelniederhammer — CHANGES_REQUESTED** [review]
Did you forget to commit the other image, too?

File: lapis2-docs/src/content/docs/concepts/response-format.mdx (hunk 1 above)
Comment (JonasKellerer) [reply lapis2-docs/src/content/docs/concepts/response-format.mdx:29]: I added a ticket. https://github.com/GenSpectrum/LAPIS/issues/644

**Review by fengelniederhammer — APPROVED** [review]
LGTM

**Pull request merged**